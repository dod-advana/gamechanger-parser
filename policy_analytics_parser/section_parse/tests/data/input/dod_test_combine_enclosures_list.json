{
    "filename": "DoDI 1234.06",
    "doc_type": "DoDI",
    "all_sections": [
        ["Enclosures "],
        ["1.  References "],
        ["2.  Responsibilities "],
        ["3.  Procedures "],
        ["Glossary "],
        ["ENCLOSURE 1"],
        ["REFERENCES"]
    ]
}
