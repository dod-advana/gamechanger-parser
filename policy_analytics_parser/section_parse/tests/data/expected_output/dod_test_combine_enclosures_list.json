{
        "all_sections": [
        [
            "Enclosures ",
            "1.  References ",
            "2.  Responsibilities ",
            "3.  Procedures ",
            "Glossary "
        ],
        ["ENCLOSURE 1"],
        ["REFERENCES"]
    ]
}
